languages/java/exercises/concept/conditionals-ternary/.docs/after.md

```diff
@@ -0,0 +1,52 @@
+## Ternary Operator
+
+The _ternary operators_ can be thought of as being a compact version of _if-else_. It's usually used in (but not restricted to) return statements, it needs just one single to make the decision, returning the left value if the expression is `true` and the right value if `false`.
+
+A lot of simple _if/else_ expressions can be simplified using _ternary operators_
+
+```java
+
+// this ternary statement
+return 5 > 4 ? true : false;
+
+// is equivalent to this if/else
+if ( 5 > 4 ) {
+    return true;
+} else {
+    return false;
+}
+```
+
+So, how to decide between _if-else_ and _ternary_ ? Well, _Ternary_ is used in simple cenarios, where you just need to return a value based on a condition and no extra computation is needed, use _if-else_ for everthing else, like nested conditions, big expressions and when more than one line is needed to decide the return value.
+
+And don't abuse of nested _ternary operators_ as the expression becomes confusing, prefer nested if's to handle those cases.
```

Comment:
Maybe rephrase this slightly? Abuse sounds a bit strong IMHO.

```suggestion
While you can nest _ternary operators_, the code often becomes hard to read. In these cases, nested if's are preferred.
```

Reply:
of course, hahha